README.md

```diff
@@ -44,8 +44,70 @@ The keypairs are RSA with 2048 bit private keys.
       signed with the "tpm" key. A UKI signed with the "tpm" key permit access to
       the TPM password for administration.
 
+# Entities / Ownership
+**Michael** (think microsoft) owns the firmware.
+  * owns keys: uefi-pk, uefi-kek
+  * public keys for uefi-pk and uefi-kek are the UEFI Platform key and UEFI Key Exchange Key respectively.
+  * In secure boot mode, the firmware measures into PCR-7 the signer of an EFI application.
+
+**Mikey** owns a key that is in the system's UEFI DB.
+  * owns keys: uefi-db
+  * uefi-db here is a single key that is present in the uefi-db.
+  * EFI applications signed by uefi-db will be executed by the UEFI in SecureBoot mode.
+  * Mikey signs Steven's shim with Mikey's uefi-db key.
+
+**Steven** owns the shim.efi application:
+  * owns keys: uki-limited, uki-production, uki-tpm
```

Comment:
I don't think this follows.

If Steven owns the shim.efi application, then he puts the certificates for uki-limited etc in the shim.  But that doesn't mean that he has access to the corresponding private keys.